FlowersStoreTests/TestHelpers/VerifyMethodWrapper/ManagerVerifyMethodProtocol.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ManagerVerifyMethodProtocol.swift
+//  ShoestockTests
+//
+//  Created by Leonir on 29/10/21.
+//  Copyright © 2021 Netshoes. All rights reserved.
+//
+
+import Foundation
+
+protocol ManagerVerifyMethodProtocol {
+    associatedtype T
+    
+    var count: Int { get }
+    var args: [T] { get }
+    var methodName: String { get }
+    
+    func getDescribeArgs() -> String?
+    
+    /// Verify if a method was called one time
+    /// - Parameters:
+    ///   - file: The class file
+    ///   - line: The method class line
+    /// - Returns: Return `true` if it was called - result is discartable
+    func wasCalledOnce(
+        file: StaticString,
+        line: UInt
+    ) -> Bool
+    
+    /// Verify if a method was called `count` time
+    /// - Parameters:
+    ///   - count: The number of times to check if a method has been called
+    ///   - file: The class file
+    ///   - line: The method class line
+    /// - Returns: Return `true` if it was called accord `count` times - result is discartable
+    func wasCalled(
+        count: Int,
+        file: StaticString,
+        line: UInt
+    ) -> Bool
+    
+    /// Verify if a method was never called
+    /// - Parameters:
+    ///   - describeArguments: String to describe a context failed message
+    ///   - file: The class file
+    ///   - line: The method class line
+    func wasNeverCalled(
+        file: StaticString,
+        line: UInt
+    )
+}
+
+extension ManagerVerifyMethodProtocol {
+    @discardableResult
+    func wasCalledOnce(
+        file: StaticString = #file,
+        line: UInt = #line
+    ) -> Bool {
+        return TestHelpers.verifyMethodCalledOnce(
+            methodName: methodName,
+            callCount: count,
+            describeArguments: getDescribeArgs(),
+            file: file,
+            line: line
+        )
+    }
+    
+    @discardableResult
+    func wasCalled(
```

Comment:
<!--
  0 failure: 
  1 warning:  Function 'wasCall...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStoreTests/TestHelpers/VerifyMethodWrapper/ManagerVerifyMethodProtocol.swift;
  Line: 69;
-->

- :warning: Function 'wasCalled(count:file:line:)' is unused